cocos/core/animation/deprecated.ts

```diff
@@ -1,63 +0,0 @@
-/*
- Copyright (c) 2020 Xiamen Yaji Software Co., Ltd.
-
- https://www.cocos.com/
-
- Permission is hereby granted, free of charge, to any person obtaining a copy
- of this software and associated engine source code (the "Software"), a limited,
- worldwide, royalty-free, non-assignable, revocable and non-exclusive license
- to use Cocos Creator solely to develop games on your target platforms. You shall
- not use Cocos Creator software for developing other software or tools that's
- used for developing games. You are not granted to publish, distribute,
- sublicense, and/or sell copies of Cocos Creator.
-
- The software or tools in this License Agreement are licensed, not sold.
- Xiamen Yaji Software Co., Ltd. reserves all rights not expressly granted to you.
-
- THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
- IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
- FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
- AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
- LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
- OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
- THE SOFTWARE.
- */
-
-/**
- * @packageDocumentation
- * @module animation
- */
-
-import { replaceProperty } from '../utils/x-deprecated';
-import { Animation } from './animation-component';
-import { AnimationClip } from './animation-clip';
-import { js } from '../utils/js';
-import { legacyCC } from '../global-exports';
-
-// deprecated
-replaceProperty(Animation.prototype, 'Animation', [
-    {
-        name: 'getAnimationState',
-        newName: 'getState',
-    },
-    {
-        name: 'addClip',
-        newName: 'createState',
-    },
-    {
-        name: 'removeClip',
-        newName: 'removeState',
-        customFunction (...args: any) {
-            const arg0 = args[0] as AnimationClip;
-            return Animation.prototype.removeState.call(this, arg0.name);
-        },
-    },
-]);
-
-/**
- * Alias of [[Animation]]
- * @deprecated Since v1.2
- */
-export { Animation as AnimationComponent };
-legacyCC.AnimationComponent = Animation;
-js.setClassAlias(Animation, 'cc.AnimationComponent');
```

Comment:
This kind of deprecation wasn't notifying user clearly, please first make the deprecation warning very clear in this version, then remove it in v3.6